enterprise/metrics/src/main/java/org/neo4j/metrics/source/PullRequestMetric.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2002-2016 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.metrics.source;
+
+import java.util.concurrent.atomic.AtomicLong;
+
+import org.neo4j.coreedge.catchup.tx.edge.PullRequestMonitor;
+
+public class PullRequestMetric implements PullRequestMonitor
+{
+    private AtomicLong lastRequestedTxId = new AtomicLong( 0 );
```

Comment:
I think I can't use it there as we are setting the value to a specific value rather than incrementing/decrementing. 

We can change `events` to use a `LongAdder` though, will do that